index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <nav class="menu">
+        <h1 class="logo">MIRNA</h1>
+        <img src="./image/Enabled.png" alt="hamburger icon" class="hamburger_icon">
+    </nav>
+    <header class="header">
+        <h2 class="header_title">I'm Mirna, welcome to my portfolio!</h2>
+        <p>I'm a software developer!
+             I can help you build a product ,
+              feature or website Look through 
+              some of my work and experience! 
+              If you like what you see and have 
+              a project you need coded, 
+              don't hestiate to contact me.</p>
+        <h3 class="final_msj">LET'S CONNECT</h3>
+        <ul class="header_list">
+            <li class="header_icon"><img src="./image/twitter_icon.png" alt="twitter_icon"></li>
+            <li class="header_icon"><img src="./image/Linkedin icon.png" alt="linkedin_icon"></li>
+            <li class="header_icon"><img src="./image/m_icon.png" alt="m_icon"></li>
+            <li class="header_icon"><img src="./image/github_icon.png" alt="github"></li>
+            <li class="header_icon"><img src="./image/V_icon.png" alt="V_icon"></li>
+        </ul>
+    </header>
```

Comment:
- [ ] Please wrap up your code with `section` instead of `header.` It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.